js/ui/collection/ui.collection_widget.base.js

```diff
@@ -400,6 +404,10 @@ const CollectionWidget = Widget.inherit({
         }
     },
 
+    _isDisabled($el) {
```

Comment:
Here is the same naming issue I mentioned above :)

Reply:
Sorry, wanna sleep :D Fixed here [ef0c18fa47f75bf7ad928610beee9263719f13ab].